c++/g2.cpp

```diff
@@ -41,7 +41,7 @@ namespace pomerol2triqs {
       o     = -o;
     }
 
-    std::complex<double> res = (sqrt(2 * l + 1) / sqrt(o)) * std::pow(1_j, o + l) * boost::math::cyl_bessel_j(l + 0.5, o * pi / 2);
+    std::complex<double> res = (sqrt(2 * l + 1) / sqrt(o)) * std::pow(1i, o + l) * boost::math::cyl_bessel_j(l + 0.5, o * pi / 2);
```

Comment:
g++ complains:
```
/home/travis/build/krivenko/pomerol2triqs/c++/g2.cpp: In function ‘std::complex<double> pomerol2triqs::t_bar(int, int)’:

/home/travis/build/krivenko/pomerol2triqs/c++/g2.cpp:44:80: error: no matching function for call to ‘pow(__complex__ int, int)’

     std::complex<double> res = (sqrt(2 * l + 1) / sqrt(o)) * std::pow(1i, o + l) * boost::math::cyl_bessel_j(l + 0.5, o * pi / 2);

                                                                                ^
```

I guess you have to put `using namespace std::complex_literals;` somewhere.

Reply:
The `using namespace` directive is already done in the TRIQS header that previously defined the operator `_j`.
This seems to rather be an issue with g++-7. This compiles just fine for clang and g++-8 and up, see [here](https://godbolt.org/z/rqP43G). Using `std::pow(std::complex<double>{1i}, o+l)` seems to work around this.
Do you want me to put this?

Reply:
Since we want to target Ubuntu Bionic, which ships GCC 7, there should be a workaround.